plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -9073,5 +9073,44 @@ body.woocommerce_page_wc-settings {
 			margin-bottom: 0;
 		}
 	}
+
+	/**
+	 * Default for footer on WooCommerce settings pages.
+	 */ 
+	#wpfooter {
+		position: relative;
+	}
+}
+
+/**
+ * Fix for footer on various settings pages.
+ */ 
+body.woocommerce-admin-page__customers,
+body.woocommerce-settings-payments-tab,
+body.woocommerce_page_wc-settings-shipping-options,
+body.woocommerce_page_wc-settings-shipping-classes,
+body.woocommerce_page_wc-settings-integration,
+body.woocommerce_page_wc-settings-site-visibility,
+body.woocommerce_page_wc-settings-point-of-sale,
+body.woocommerce_page_wc-settings-advanced-keys,
+body.woocommerce_page_wc-settings-advanced-webhooks,
+body.woocommerce_page_wc-settings-advanced-legacy_api,
+body.woocommerce_page_wc-settings-advanced-woocommerce_com,
+body.woocommerce_page_wc-settings-advanced-blueprint,
+body.woocommerce_page_wc-settings-products-download_urls,
+body.woocommerce_page_wc-settings-products-advanced {
+	#wpfooter {
+		position: absolute;
+	}
 }
 
+/**
+ * Hide screen meta links on Advanced Keys settings page.
+ */ 
+body.woocommerce_page_wc-settings-advanced-keys,
+body.woocommerce_page_wc-settings-advanced-webhooks,
+body.woocommerce-admin-page__extensions {
+	#screen-meta-links {
+		display:none;
+	}
+}
```

Comment:
AFAICT, the "Help" or "Screen Options" sections aren't needed on these routes, and they broke the layout. Not specifically related to the reported issue, but thought I'd clean this up since I saw them.